### ISSUE TITLE
Execution after Failcheck finds NAN

### ISSUE THREAD

**Issue by llaniewski** [CFD-GO/TCLB#49]
It would be nice if we could execute something still after the failcheck finds NANs.

I propose this syntax:
```xml
<Failcheck>
  <VTK name="withnan" what="ALL"/>
</Failcheck>
```

This would write the VTK file after finding the nan. This could help identify that is happening better. I thinks it can be done with something like ExecuteInternal().

**Comment by shkodm** [comment]
Yea, it is really good idea. I will implement it, but the "syntax" you proposed is not visible...

**Comment by llaniewski** [comment]
@shkodm I fixed the syntax

**Comment by llaniewski** [comment]
Closed by #54
